ShellPkg/Application/Shell/ConsoleLogger.c

```diff
@@ -719,13 +719,27 @@ ConsoleLoggerPrintWithPageBreak (
   CONST CHAR16  *LineStart;
   CHAR16        *StringCopy;
   CHAR16        TempChar;
+  INT32         curCol;    // MU_CHANGE

```

Comment:
Sorry, to request a change for something so trivial, but can you please update this to match naming conventions `CurCol`?